apiserver/pkg/util/config.go

```diff
@@ -13,7 +13,8 @@ const (
 	RayClusterUserLabelKey            = "ray.io/user"
 	RayClusterVersionLabelKey         = "ray.io/version"
 	RayClusterEnvironmentLabelKey     = "ray.io/environment"
-	RayServiceLabelKey                = "ray.io/service"
+	RayOriginatedFromNameLabelKey     = "ray.io/originated-from-name"
```

Comment:
`ray.io/owner-name` might be better?

Reply:
We have considered using `owner-name`. The question is that RayJob/RayService owns RayCluster, and RayCluster owns Ray Pods. We may need to associate the Ray Pods from both the RayJob/RayService level and the RayCluster level.

We don't have a good idea for the label name to represent the "root" of the owner/dependent tree. Do you have any good ideas?

Reply:
> We don't have a good idea for the label name to represent the "root" of the owner/dependent tree. Do you have any good ideas?

Would it be simpler to just have a single label `ray.io/originiated-from` that contains both the type and name?
```
ray.io/originated-from: rayjob/myjob
ray.io/originiated-from: raycluster/mycluster
ray.io/originiated-from: rayservice/myservice
```

Reply:
It sounds good to me. cc @rueian WDYT?

Reply:
That is a great idea. However, k8s will reject `/` in label value with an error like this:
```
metadata.labels: Invalid value: "rayjob/myjob": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')
```
How about using `.` instead?

```
ray.io/originated-from: rayjob.myjob
ray.io/originated-from: raycluster.mycluster
ray.io/originated-from: rayservice/.myservice
```

Reply:
Is it possible for CR name to have `.`? If it is impossible, using `.` makes sense to me.

Reply:
Unfortunately, `.` is allowed in resource names, but `_` is not.
So we are safe to use the following:
```
ray.io/originated-from: rayjob_myjob
ray.io/originated-from: raycluster_mycluster
ray.io/originated-from: rayservice_myservice
```

Reply:
> Unfortunately, `.` is allowed in resource names, but `_` is not.

The use of `_` sounds good to me. What's the regex for resource names?

Reply:
`